modern-business/sass/blocks/_blocks.scss

```diff
@@ -1,7 +1,9 @@
 /* !Block styles */
 
 .entry .entry-content > *,
-.entry .entry-summary > * {
+.entry .entry-summary > * 
```

Comment:
is the comma also an issue here?

Reply:
that is where the comma was missing, added now - you must have been looking at the previous commit

Reply:
very strange, not sure why github hadn't caught up to that 😆 apologies!